lib/tokensecurity.js

```diff
@@ -618,7 +617,8 @@ module.exports = function (app, config) {
     }
 
     if (!token) {
-      token = req.cookies && req.cookies.JAUTHENTICATION
+      token =
+        req.cookies && (req.cookies.SK_COOKIE || req.cookies.JAUTHENTICATION)
```

Comment:
Why the additional cookie name? How about making it configurable?

Reply:
SK_COOKIE was in the current PR that's open in specification. It is no longer there, so please revert this change. (or make it configurable)

Reply:
Okay, it's gone